src/repositories/UserRepository.ts

```diff
@@ -29,16 +29,16 @@ export class UserRepository extends AbstractRepository<UserModel> {
       .getOne();
   }
 
-  public async savePost(user: UserModel, post: PostModel): Promise<PostModel | undefined> {
-    user.saved.push(post)
-    await this.repository.save(user)
-    return post
+  public async savePost(user: UserModel, post: PostModel): Promise<PostModel> {
+    user.saved.push(post);
```

Comment:
i don't see semicolons anywhere else in the code, we need to be consistent with this

Reply:
The end of all lines should have one. I think that's consistent throughout if we're counting the querybuilder parts as one line